### DISCUSSION TITLE
SUI Ecosystem

### DISCUSSION THREAD

**Discussion by Rohman22** [MystenLabs/sui#7446] [General]
What do you thing about SUI?

**Comment by SquareRut** [comment]
Sui is another big name that's gonna be heard in the crypto industry....they have a lot to offer

**Reply by Kessyboy** [reply]
its gonna cook

**Reply by JoeVee1k** [reply]
What SUI is cooking is the biggest in Crypto history.

**Comment by skeap79** [comment]
Sui is backed by Mysten Labs - founded by former senior executives at Meta’s Novi Research. They are more focusing on speed, scalability and it's unique approach to security.

**Comment by thosincul** [comment]
I think is going to be a best ecosystem in the crypto world

**Reply by Ghazwan697** [reply]
I totally agree
